### PR TITLE
demo additional and small fix

### PR DESCRIPTION
**Pull request by 7dir**
- add mouse-cursor-component
- remove a-aframe from dist
- buttonCustom (remove duplicate id) switch stats on/off
- remove examples/node_modules

### CODE REVIEW

**Comment by caseyyee** [comment]
Thanks for the PR 🎉

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .DS_Store
 node_modules
```
Comment (7dir) [inline .gitignore]: examples/node_modules???

Comment (7dir) [inline .gitignore]: ~926 files?

Comment (7dir) [inline .gitignore]: like this?
https://github.com/demo3d/aframe-ui-widgets/blob/master/.gitignore

Comment (caseyyee) [inline .gitignore]: Yeah the 926 files.  Just having node_modules in there should omit anything with that directory name.   Maybe have to try with a trailing `/`?

Comment (7dir) [inline .gitignore]: storeing node_modules in git bad idea and bad practice

```sh
git clone yours_git_prj.git
cd yours_git_prj.git
yarn # or npmc i or npm i
# then
npm run dev
```

`examples/node_modules` for what? `budo`?

**Comment by caseyyee** [comment]
Thank you!

**Pull request merged**